components/Home/MainBanner.js

```diff
@@ -0,0 +1,193 @@
+import React, { useState } from "react";
+import Link from "next/link";
+import FsLightbox from "fslightbox-react";
+
+export const BOOKING_LINK = "https://cal.com/shayan-salehis-btb8mk/30min";
+const styles = {
+  banner: {
+    left: "100px",
+    position: "relative",
+    width: "400px",
+    height: "400px",
+    borderRadius: "100px",
+    overflow: "hidden",
+  },
+  overlay: {
+    position: "absolute",
+    top: 0,
+    left: 0,
+    width: "100%",
+    height: "100%",
+    background: "rgba(0, 0, 0, 0.2)", // Adjust opacity here
+    zIndex: 1,
+  },
+  background: {
+    width: "100%",
+    height: "100%",
+    opacity: "0.5",
+    objectFit: "cover",
+    zIndex: -10,
```

Comment:
this is the desired design for how it blends to the background.
https://drive.google.com/file/d/1JOMmk01skQTtlC7cRI2nvDajPuKJLvlS/view

I don't know how to do this but some kind of "radial gradient" sounds right. See https://stackoverflow.com/questions/32059991/how-to-make-a-radial-gradient-with-transparency-in-css